components/button/src/overrides/button.js

```diff
@@ -1,11 +1,76 @@
 /** @jsx jsx */
 
-import { jsx, useBrand, asArray, useMediaQuery, getLabel } from '@westpac/core';
+import { jsx, useBrand, asArray, useMediaQuery, classNames } from '@westpac/core';
 import { forwardRef } from 'react';
 
+// ==============================
+// Component
+// ==============================
+
 const Button = forwardRef(({ state: { tag: Tag }, ...rest }, ref) => <Tag ref={ref} {...rest} />);
 
-const buttonStyles = (_, { look, size, soft, block, justify, disabled }) => {
+// ==============================
+// Styles
+// ==============================
+const blenderStyles = (_, { look, size, soft, block, justify, disabled }) => {
```

Comment:
@dominikwilkowski Here's my attempt of diff'ing modifier styles with the base styles from default props. Is working correctly for me with the blender. Thoughts?